subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugs.java

```diff
@@ -328,12 +328,21 @@ public FileTree getSource() {
     @SkipWhenEmpty
     @PathSensitive(PathSensitivity.RELATIVE)
     @InputFiles
+    protected FileCollection getCandidateClassFiles() {
+        return getClasses().getAsFileTree();
```

Comment:
How is this now different to annotating `getClasses` directly? If we do not filter by `*.class` then I would actually not add this new method but just use the old method as is and only change the convention mapping.